.gitignore

```diff
@@ -33,6 +33,12 @@ node_modules
 # Optional REPL history
 .node_repl_history
 
+# No NPM config to pass CFS compliance
+.npmrc
```

Comment:
Confused about why we are ignoring lock files and .npmrc... Just like nuget.config we need to have a `.npmrc` at the root

Reply:
Not according to the task - I removed .npmrc and it passes.